### DISCUSSION TITLE
Discussion forum setup

### DISCUSSION THREAD

**Discussion by Renuka-S** [Sunbird-Lern/Community#62] [Documentation] [answered]
Hi Team, 

Could you please redirect me to the discussion forum setup & configuration documentation & its user documentation?

**Comment by reshmi-nair** [comment, marked as answer]
@Renuka-S Pls find the necessary links to setup and configure discussion forum
[Installation Guide](https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide)
[Product-and-developer-guide](https://lern.sunbird.org/learn/product-and-developer-guide/discussion-forum)

**Comment by Renuka-S** [comment]
@reshmi-nair Thank you for this super helpful. 
Could you also help me with which SB version discussion has been added? We are on a lower version so this information is needed.

**Reply by reshmi-nair** [reply]
@Renuka-S It was added in release-4.2.0 of SB version.

**Reply by Renuka-S** [reply]
@reshmi-nair Thank you for confirmation. cc: @gandham-santosh @komalmane
